backends/transforms/remove_clone_ops.py

```diff
@@ -48,3 +55,29 @@ def call(self, graph_module: torch.fx.GraphModule) -> PassResult:
         graph_module.recompile()
         dead_code_elimination_pass(graph_module)
         return PassResult(graph_module, True)
+
+    def aten_clone_is_non_identity(self, node: torch.fx.Node) -> bool:
```

Comment:
Let's combine the two check functions, `aten_clone_is_non_identity ` and `_clone_dim_order_is_non_identity ` into a single function (maybe called `_is_non_identity_clone`). Under current scenario we will always use the funcs together and this func should be private.